### ISSUE TITLE
Pause and resume downloads?

### ISSUE THREAD

**Issue by Danielx64** [Kalanyr/gogrepoc#30]
Hi there,

Will this let you pause and resume downloads? In some cases I can't finish downloading in one session and I would like to pause and resume.

Cheers

**Comment by Kalanyr** [comment]
Yes, if you terminate the program while it's downloading , the downloads
will resume from where they left off if you run the download command again
(at the moment if you run a download command with different parameters it
will delete anything partially downloaded and not part of the new command
but that's on my to do list)

**Comment by Danielx64** [comment]
Awesome thank you, and last question (for now), When will support for downloading movies come? Trying to download that 20GB movie (2014 The Witcher 3: Wild Hunt concert (4K) is giving me grief right now)......

**Comment by Kalanyr** [comment]
It's in my current in progress build, so probably in the next couple of
months (Persona 5 Royal is occupying my free time at the moment).
